code/mob/wraith.dm

```diff
@@ -606,6 +606,7 @@
 		boutput(W, "<B>You are a wraith! Terrorize the mortals and drive them into releasing their life essence!</B>")
 		boutput(W, "Your astral powers enable you to survive one banishment. Beware of salt.")
 		boutput(W, "Use the question mark button in the lower right corner to get help on your abilities.")
+		boutput(W, "please dm retkid#9135 for any sound-related feedback")
```

Comment:
unnecessary imo

Reply:
To you maybe. However I've never played Wraith to a degree to learn the meta. There is a good chance that I missed something that breaks the meta for some reason. I don't think there is but I want to be sure. Or maybe a sound gets repetitive (you'll notice I have less Rands, mostly because the timers are longer so I need less sounds) and they'd like more variability.  And I'd like any issues to me my accountability. 

Maybe I accidentally put a sound too loud who knows.